Functions/FirebaseFunctions/FIRFunctions.m

```diff
@@ -149,12 +149,12 @@ - (instancetype)initWithProjectID:(NSString *)projectID
 }
 
 - (void)useLocalhost {
-  [self useEmulatorWithHost:@"http://localhost" port:5005];
+  [self useEmulatorWithHost:@"localhost" port:5005];
 }
 
 - (void)useEmulatorWithHost:(NSString *)host port:(NSInteger)port {
   NSAssert(host.length > 0, @"Cannot connect to nil or empty host");
-  NSString *origin = [NSString stringWithFormat:@"%@:%li", host, (long)port];
+  NSString *origin = [NSString stringWithFormat:@"http://%@:%li", host, (long)port];
```

Comment:
Yeah, it is a breaking change. I'm a bit hesitant to push this change as is, especially taking into account that the bug is described as intended behaviour in the [docs](https://firebase.google.com/docs/functions/local-emulator#ios---swift). Maybe we can make it non-breaking by adding the http prefix conditionally, e.g. 
```
NSString *prefix = [host hasPrefix:@"http"] ? @"http://" : @"";
NSString *origin = [NSString stringWithFormat:@"%@%@:%li", prefix, host, (long)port];
```
In this case both the intended behaviour and the old workaround would work. We can remove the workaround at Firebase 8 where we intend to do other breaking changes (if decided so please crate a ticket). WDYT?

Reply:
Those docs reference the old `.useFunctionsEmulator(origin: "http://localhost:5001")`, this PR only affects the newer `useEmulator("localhost", 5001)` function.

But I agree we can make this non-breaking.

Reply:
Ah, true, sorry, I misread the snippet.

Reply:
Ok I implemented your change, thank you!